package.json

```diff
@@ -118,7 +118,6 @@
     "jest": "^26.6.3",
     "jest-transform-stub": "^2.0.0",
     "lint-staged": "^11.1.1",
-    "pinia": "^2.0.11",
```

Comment:
I'm confused by this change.

Reply:
I would also be confused if I saw it this way :) pinia is both in dependencies and devDependencies, so I removed it from the latter.